vispy/ipython.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""Entry point for vispy's IPython bindings"""
+
+import IPython
```

Comment:
same, nest this import, put inside `try/except`, and in fact try to make this code more DRY with the testing code. Maybe put IPython version checking function in `vispy/util` somewhere, and import that both from `testing` and from here.